### PR TITLE
Smallish Visual Studio updates to high page view topics

### PR DESCRIPTION
**Pull request by gewarren**
- replace ASP.NET 5 with ASP.NET Core
- add "or later" after VS version
- whitespace cleanup (diff with ignore whitespace on)

Part of the fix for #7172.

### CODE REVIEW

File: docs/framework/whats-new/index.md [hunk 1]
```diff
@@ -370,7 +370,7 @@ These methods return an empty enumerable unless <xref:System.Windows.Diagnostic
...
+A diagnostic assistant can now receive a notification whenever a [StaticResource](../wpf/advanced/staticresource-markup-extension.md) reference is resolved. (The feature is for use by diagnostic assistants, not by production applications.) A diagnostic assistant such as Visual Studio’s “Edit-and-Continue” facility may want to update all uses of a resource when its value in a <xref:Windows.UI.Xaml.ResourceDictionary> changes. WPF does this automatically for [DynamicResource](../wpf/advanced/dynamicresource-markup-extension.md) references, but it intentionally does not do so for [StaticResource](../wpf/advanced/staticresource-markup-extension.md) references. Starting with the .NET Framework 4.7.2, the diagnostic assistant can use these notifications to locate those uses of the static resource.
 
 The notification is implemented by the new <xref:System.Windows.Diagnostics.ResourceDictionaryDiagnostics.StaticResourceResolved?displayProperty=nameWithType> event:
```
Comment (rpetrusha) [inline docs/framework/whats-new/index.md]: unrelated to your PR: The ClickOnce link is not working. could you remove blank line 402?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for making these changes, @gewarren. I've left a number of comments, many of them unrelated to your PR but that would be nice to address.

File: docs/framework/whats-new/index.md [hunk 2]
```diff
@@ -398,9 +403,7 @@ public class StaticResourceResolvedEventArgs : EventArgs
 
 The event is not raised (and its `add` accessor is ignored) unless <xref:System.Windows.Diagnostics.VisualDiagnostics> is enabled and the [`ENABLE_XAML_DIAGNOSTICS_SOURCE_INFO`](xref:System.Windows.Diagnostics.VisualDiagnostics.GetXamlSourceInfo%2A) environment variable is set.
 
-<a name="clickonce472" />
-
-#### ClickOnce
+#### <a name="clickonce472" /> ClickOnce
```
Comment (rpetrusha) [inline docs/framework/whats-new/index.md]: The link still isn't working. I think it needs to be on two separate lines as it originally was:

\<a name="clickonce472" />
\#\#\#\# ClickOnce

File: docs/framework/whats-new/index.md [hunk 3]
```diff
@@ -1214,7 +1226,7 @@ SqlClient now automatically provides faster connections to an AlwaysOn Availabil
 
          ASP.NET now provides a simple task-based API for asynchronous response flushing, <xref:System.Web.HttpResponse.FlushAsync%2A?displayProperty=nameWithType>, that allows responses to be flushed asynchronously by using your language's `async/await` support.
 
-    - `Model binding supports task-returning methods`
+    - **`**Model binding supports task-returning methods**
```
Comment (rpetrusha) [inline docs/framework/whats-new/index.md]: This is badly formatted -- the opening \*\*\` can be removed.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @gewarren. Just two comments to address.

**Comment by gewarren** [comment]
@rpetrusha Thanks so much for your reviews. The ClickOnce bookmarks are both working now and I think this is good to go.

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @gewarren. This looks good, so I'll merge it now.

**Review by rpetrusha — APPROVED** [review]

**Comment by gewarren** [comment]
@rpetrusha Thanks very much Ron!

**Pull request merged**